pkg/build/pipelines/goreleaser/build.yaml

```diff
@@ -0,0 +1,83 @@
+name: Run a build using the GoReleaser
+
+needs:
+  packages:
+    - busybox
+    - ca-certificates-bundle
+    - goreleaser
+
+inputs:
+  args:
+    description: |
+      List of space-separated args to pass to the GoReleaser `release` command.
+    required: false
+  
+  output:
+    description: |
+      Filename to use when writing the binary. The final install location inside
```

Comment:
I would rather the `${{targets.contextdir}}/usr/bin` prefix be explicit.  e.g.

```yaml
pipeline:
  - uses: goreleaser/build
    with:
      output: ${{targets.contextdir}}/usr/bin/foo
```

Incidentally, I want to refactor `go/build` to work that way too.  See #461.

Reply:
That makes a lot of sense, thanks, but if you don't mind I'd like to ask a question. How do we ensure that the `foo` is the binary name or the directory in the example above? Because we need to create a directory where we put the binary, so, we need to determine both the directory and the binary name.